### ISSUE TITLE
Fluid.Storage: Heat Transfer models

### ISSUE THREAD

**Issue by PMehrfeld** [RWTH-EBC/AixLib#345]
I came across some questions regarding modelling style:

1. `AixLib.Fluid.Storage.BaseClasses.PartialHeatTransferLayers`: Is it really necessary to declare `n, Medium` and `data` as `outer`? Especially since these are commonly used names.
2. `AixLib.Fluid.Storage.BaseClasses.HeatTransferLambda...`: These 3 models use the variable `beta`. For me it looks like they can be declared as parameters instead of variables.

@SebastianStinner: Do you maybe have answers to these two questions?

**Comment by SebastianStinner** [comment]
@PMehrfeld : 
1. I think there was a reason to do this, but we could try to change that.
2. This is possible, except if we see` beta` as a constant which should not be changed from the modeller.

**Comment by PMehrfeld** [comment]
1. I deleted the prefixes.
2. I converted the variables into parameters.

Furthermore, I removed some `inner` prefixes, like in `AixLib.Fluid.Storage.BufferStorage.Medium`, `AixLib.Fluid.Storage.BaseClasses.HeatingCoil` and `AixLib.Fluid.Storage.BufferStorage`. 
I also passed the parameters declared in `BufferStorage` through, since they are not using the `outer\inner` concept anymore.

Example still works and produces same results in terms of heat loss.
